src/Mod/Start/Gui/ThemeSelectorWidget.cpp

```diff
@@ -135,7 +154,8 @@ bool ThemeSelectorWidget::eventFilter(QObject* object, QEvent* event)
 void ThemeSelectorWidget::retranslateUi()
 {
     _titleLabel->setText(QLatin1String("<h2>") + tr("Theme") + QLatin1String("</h2>"));
-    _descriptionLabel->setText(tr("More themes are available online using the Addon Manager"));
+    _descriptionLabel->setText(tr("More themes are available online using the ")
+                               + QLatin1String("<a href='freecad:Std_AddonMgr'>Addon Manager</a>"));
```

Comment:
Addon Manager does actually get translated (in at least some languages), so it should be in the translation string. It's not ideal to have HTML in there, but I think that's the only really good cross-language way of handling it.

Reply:
I've changed the text to be exactly the same as in settings. Hopefully this'll the job of translators at least a bit easier.